### PR TITLE
update notopen projects to use nuget package reference

### PR DESCRIPTION
**Pull request by blounsbury-usbr**

### CODE REVIEW

File: MigrationBackup/2bdf37c9/Reclamation.TimeSeries.Excel/NuGetUpgradeLog.html
Comment (ktarbet) [inline MigrationBackup/2bdf37c9/Reclamation.TimeSeries.Excel/NuGetUpgradeLog.html:1]: Probably don't want this file in the commit?

**Review by ktarbet — CHANGES_REQUESTED** [review]
There are a few UpgradeLog.html files that should not be part of the PR.
We should also be moving to the SDK style project files, and moving away from packages.config files.

**Comment by ktarbet** [comment]
When I was working on updating the other projects there was a Right click menu: 
```
right-click on the References node or the packages.config file and select Migrate packages.config to PackageReference...
```
https://learn.microsoft.com/en-us/nuget/consume-packages/migrate-packages-config-to-package-reference

**Comment by blounsbury-usbr** [comment]
Yep, that's exactly what I did, right-clicked on packages.config.

File: MigrationBackup/2bdf37c9/Reclamation.TimeSeries.Excel/NuGetUpgradeLog.html
Comment (ktarbet) [reply MigrationBackup/2bdf37c9/Reclamation.TimeSeries.Excel/NuGetUpgradeLog.html:1]: Ok. sorry about putting those files in.   Would you like me to approve the PR as is?

Comment (blounsbury-usbr) [reply MigrationBackup/2bdf37c9/Reclamation.TimeSeries.Excel/NuGetUpgradeLog.html:1]: No worries. I'll delete the MigrationBackup folder from the branch first.

**Review by ktarbet — APPROVED** [review]

**Pull request merged**